### PR TITLE
Add PT-BR automatic translation redirect.Rmd

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: redirect.pt.Rmd [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# How to set a redirect {#redirect}
+# Como definir um redirecionamento {#redirect}
 
-## Non GitHub pages site (e.g. Netlify) {#non-git-hub-pages-site-e-g-netlify}
+## Site que não seja de páginas do GitHub (por exemplo, Netlify) {#non-git-hub-pages-site-e-g-netlify}
```
Comment (IldySilva) [inline redirect.pt.Rmd:3]:
```suggestion
## Site que não seja do github pages (por exemplo, Netlify) {#non-git-hub-pages-site-e-g-netlify}
```

File: redirect.pt.Rmd [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-Replace the content of the current website with a `index.html` and `404.html` files both containing:
+Substituir o conteúdo do site atual por um site `index.html` e `404.html` contendo ambos os arquivos:
```
Comment (IldySilva) [inline redirect.pt.Rmd:5]:
```suggestion
Substitua o conteúdo do site atual por um arquivos com o nome `index.html` e um com nome  `404.html` contendo ambos os arquivos:
```

File: redirect.pt.Rmd (hunk 1 above)
Comment (IldySilva) [inline redirect.pt.Rmd:3]:
```suggestion
## Site que não seja de páginas do Github Pages (por exemplo, Netlify) {#non-git-hub-pages-site-e-g-netlify}
```

File: redirect.pt.Rmd [hunk 3]
```diff
@@ -12,12 +12,12 @@ Replace the content of the current website with a `index.html` and `404.html` fi
...
 
-You can setup the redirect from your main user gh-pages repository:
+Você pode configurar o redirecionamento no repositório gh-pages do seu usuário principal:
 
-- create a new repository (if you don't have one yet): `https://github.com/<username>/<username>.github.io`.
-- In this repository create a directory `<pkgname>` containing 2 files: a `index.html` and `404.html` file, which both redirect to the new location (see previous subsection).
-- Test that `https://<username>.github.io/<pkgname>/index.html` now redirects.
+- crie um novo repositório (se você ainda não tiver um): `https://github.com/<username>/<username>.github.io`.
+- Nesse repositório, crie um diretório `<pkgname>` contendo 2 arquivos: um `index.html` e `404.html` que redirecionam para o novo local (consulte a subseção anterior).
+- Teste que `https://<username>.github.io/<pkgname>/index.html` agora redireciona.
```
Comment (IldySilva) [inline redirect.pt.Rmd:21]:
```suggestion
- Teste o endereço  `https://<username>.github.io/<pkgname>/index.html` que vai redirecionar.
```

**Review by IldySilva — COMMENTED** [review]
i've pull the changes : #788

**Review by yabellini — COMMENTED** [review]
Looks very good.

**Pull request merged**